IDEAS/Buildings/Components/ZoneAirModels/BaseClasses/partial_AirLeakage.mo

```diff
@@ -0,0 +1,70 @@
+within IDEAS.Buildings.Components.ZoneAirModels.BaseClasses;
```

Comment:
can you rename this to PartialAirLeakage to be consistent with A60 naming conventions?